frontend/js/marc-record-merge-postmerge-service.js

```diff
@@ -447,6 +462,29 @@ export function removeExtra035aFromMerged(preferredRecord, otherRecord, mergedRe
   };
 }
 
+export function add080VersionCode (preferredRecord, otherRecord, mergedRecordParam) {
+  const mergedRecord = new MarcRecord(mergedRecordParam);
+
+  mergedRecord.fields.filter(field => field.tag === '080')
+    .forEach(field => {
+      const hasCode2 = field.subfields.findIndex(sub => sub.code === '2');
```

Comment:
No need to save this query to a variable (We want to proceed to the next field if the subfield exists). Do this instead:
```js
if (field.subfields.some(({code}) => code === '2') {
  return;
}

Reply:
> jonollil Joni Ollila added a comment - 05.08.2019
> 
> Kysymys, onko tälläinen tilanne mahdollinen?
> 080 |a 688/689 |2 <jotain muuta tekstiä> |9 FENNI<KEEP>
> Eli code 2 sisältäisi jo jonkin muun arvon? jos tälläinen tilanne tulee halutaanko se korvata arvolla "1974/fin/fennica"
>
> ammsalme Minttu Hurme added a comment - 23.08.2019
> 
> Datassa ei tässä nimenomaisessa tapauksessa pitäis olla tapauksia, joissa kentässä olis jotain muuta sisältävä $2. Jos on niin kyseessä on jonkinlainen kuvailuvirhe - eli jos meillä olis ominaisuus, joka varoittaisi käyttäjää mahdollisesta virheestä, se vois herjata tässä. Ei tosin taida olla sellaista ominaisuutta, joten unohdetaan herja.
> 
> Yleisemmin tänkaltaisissa täydennystapauksissa ei kyllä ole syytä korvata olemassaolevaa, joten mennään sillä.
>